doc/api/sqlite.md

```diff
@@ -11,7 +11,8 @@ added: v22.5.0
 <!-- source_link=lib/sqlite.js -->
 
 The `node:sqlite` module facilitates working with SQLite databases.
-To access it:
+This feature can only be used when the `--experimental-sqlite` CLI
+flag is specified. To access it:
```

Comment:
Can you move it to the `> Stability: 1.1 - Active development` block? Similarly to what we did in e.g. https://github.com/nodejs/node/blob/f232064f35df4e10ed8363863bca704fe878012f/doc/api/globals.md#L358-L359